### PR TITLE
Add conventions, templates and CODEOWNERS

### PR DESCRIPTION
**Pull request by steffenkleinle**
:warning: DO NOT MERGE YET :warning: 

### Short description

Add conventions, templates and CODEOWNERS.

### Proposed changes

- Add section about conventions to README
- Add templates for issues and pull requests
- Add CODEOWNERS file for reviews

### Side effects

None.

### Testing

Is something missing/would you change something?
Are the github handles correct?

### Resolved issues

Fixes: None.

---

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for
+# review when someone opens a pull request.
+*       @NikoHadouken @volkramweber @steffenkleinle
```
Comment (steffenkleinle) [inline CODEOWNERS:7]: Do you know the github handle of Christian?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (steffenkleinle) [inline .github/ISSUE_TEMPLATE/bug-report.md:1]: Not sure if these issue templates are necessary or we want to have some other types of issues. Just copy/pasted. I can also remove them again.

File: CODEOWNERS (hunk 1 above)
Comment (NikoHadouken) [reply CODEOWNERS:7]: His handle is @Stift

File: README.md [hunk 3]
```diff
@@ -14,3 +14,31 @@ A few resources to get you started if this is your first Flutter project:
 For help getting started with Flutter development, view the
 [online documentation](https://docs.flutter.dev/), which offers tutorials,
 samples, guidance on mobile development, and a full API reference.
+
+## Conventions
+
+### Code Style
+
+We follow the [Effective Dart code style](https://dart.dev/effective-dart/style).
```
[the remark this answers is not in the document]
Comment (NikoHadouken) [reply README.md:22]: I read i and agree we should use it.

**Pull request closed without merging**